emission/net/ext_service/habitica/sync_habitica.py

```diff
@@ -0,0 +1,68 @@
+# Standard imports
+import json
+import requests
+import logging
+import arrow
+import attrdict as ad
+
+# Our imports
+import emission.core.get_database as edb
+import emission.net.ext_service.habitica.proxy as proxy
+import emission.analysis.result.metrics.simple_metrics as earmts
+import emission.analysis.result.metrics.time_grouping as earmt
+
+
+
+def create_habit(user_id, new_habit):
```

Comment:
No, this checks if the task has already been created, if so, it returns its id, otherwise, it creates it (see last 3 lines of method, copied below)
"response = proxy.habiticaProxy(user_id, 'POST', method_uri, new_habit)
  habit_created = response.json()
  return habit_created['data']['_id']"